skipper.yaml

```
@@ -11,6 +11,7 @@ volumes:
   - /usr/bin/kubectl:/usr/bin/kubectl
   - /usr/local/bin/kubectl:/usr/local/bin/kubectl
   - $MINIKUBE_HOME:$MINIKUBE_HOME
+  - /usr/sbin/iptables:/usr/sbin/iptables
```

```suggestion
  - $(which iptables):/usr/sbin/iptables
```
to be less dependent on the location of the hypervisor iptables binary